src/main/java/cz/it4i/fiji/datastore/security/OAuthUserEndpoint.java

```diff
@@ -0,0 +1,75 @@
+package cz.it4i.fiji.datastore.security;
+import com.google.gson.Gson;
+
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+
+@Path("/oauth-users")
+public class OAuthUserEndpoint {
+
+    @Inject
+    OAuthUserService oauthUserService;
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<OAuthUserNew> getAllOAuthUsers() {
+        return oauthUserService.getAllOAuthUsers();
+    }
+    @GET
+    @Path("/json")
```

Comment:
Consider using standard convention for forming REST API - requested type of data probably should not be part of a path.